conf/script/ha_setup

```diff
@@ -41,23 +36,13 @@ exec &> >(stdbuf -oL gawk '{ print strftime("%Y-%m-%d %H:%M:%S"), $0 }')
 post_install() {
     echo "post_install"
     # Add permisson to hacluster
-    add_permission
-
-    # Trigger logotate hourly basis
-    if [ ! -d "${LOGROTATE_DIR_DEST}" ]; then
-        mkdir -p ${LOGROTATE_DIR_DEST}
-    fi
+#    add_permission
```

Comment:
Please delete this line.

Reply:
Actually, I had commented out this line for the dev testing as I was getting "hacluster user is not available" error in VM and forgot to uncomment this line after the testing.
Now, Corrected.

Reply:
ok